test/common/crudtester.go

```diff
@@ -149,11 +149,15 @@ func (c *FederatedTypeCrudTester) createFedResource(adapter federatedtypes.FedAp
 func (c *FederatedTypeCrudTester) CheckCreate(desiredTemplate, desiredPlacement, desiredOverride pkgruntime.Object) (pkgruntime.Object, pkgruntime.Object, pkgruntime.Object) {
 	template, placement, override := c.Create(desiredTemplate, desiredPlacement, desiredOverride)
 
+	overrideResourceVersion := ""
+	if c.adapter.Override() != nil {
+		overrideResourceVersion = c.adapter.Override().ObjectMeta(override).ResourceVersion
+	}
 	c.tl.Logf("Resource versions for %s: template %q, placement %q, override %q",
```

Comment:
To prevent any confusion, I suggest re-writing the log to not include "override" if the resource doesn't contain one. Otherwise you can wait for that part of #14 to merge, which will be in a new PR.